.idea/workspace.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
i guess it should be ignored in git

update gitignore from [gitignore.io](gitignore.io) where you specify `idea` and `java` etc